Week-1/Assignment-1.2/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pesto-Assignment-1.2</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css"
+      integrity="sha512-rSWTr6dChYCbhpHaT1hg2tf4re2jUxBWTuZbujxKg96+T87KQJriMzBzW5aqcb8jmzBhhNSx4XYGA6/Y+ok1vQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/css/line-awesome.min.css"
+      integrity="sha512-vebUliqxrVkBy3gucMhClmyQP9On/HAWQdKDXRaAlb/FKuTbxkjPKUyqVOxAcGwFDka79eTF+YXwfke1h3/wfg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./css/index.css" />
+  </head>
+  <body>
+    <main>
+      <aside>
+        <i class="las la-bars"></i>
+
+        <i class="las la-home"></i>
+
+        <i class="las la-search"></i>
+
+        <i class="las la-volume-up"></i>
+
+        <i class="lar la-user"></i>
+
+        <i class="lab la-spotify"></i>
+
+        <i class="las la-cog"></i>
+
+        <i class="lab la-soundcloud"></i>
+      </aside>
+
+      <section class="hero">
+        <div class="header">
+          <img src="https://vif1g.csb.app/src/assets/cattyboard.jpg" alt="" />
+          <section class="text-left-photo">
+            <div>
+              <p>CattyBoard Top 100 Single Charts (11.06.36)</p>
+              <p>Unknown Artist</p>
+              <p>2016 . Charts . 100 songs</p>
+            </div>
+            <div>
+              <i class="las la-play"> &nbsp;Play all</i>
+
+              <i class="las la-plus">&nbsp;Add to</i>
+
+              <i class="las la-ellipsis-h">&nbsp;&nbsp;More</i>
```

Comment:
we should not use &nbsp just for indentation. This should be controlled by css, perhaps padding or margin

Reply:
Thanks, I will make relevant chnages